### ISSUE TITLE
neural cell[Obsolete]

### ISSUE THREAD

**Issue by dosumis** [obophenotype/cell-ontology#2064]
**CL term**

neural cell
http://purl.obolibrary.org/obo/CL_0002319
A cell that is part of the nervous system.

EquivalentTo: ([animal cell](https://www.ebi.ac.uk/ols4/ontologies/cl/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCL_0000548?lang=en)and[part of](https://www.ebi.ac.uk/ols4/ontologies/cl/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FBFO_0000050?lang=en)[RO](https://www.ebi.ac.uk/ols4/ontologies/ro/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FBFO_0000050)some[nervous system](https://www.ebi.ac.uk/ols4/ontologies/cl/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUBERON_0001016?lang=en)[UBERON](https://www.ebi.ac.uk/ols4/ontologies/uberon/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FUBERON_0001016))

**Suggested replacement**

This is a rare case where I think no replacement can be provided

**Reason for deprecation**

This is so general & trivial that it is useless.  In work with PCL it is leading to distracting automated classifications that are interfering with expert review.

**Comment by rays22** [comment]
I think it causes problems if there is no replacement for CL:0002319.
[CL:0002319](https://www.ebi.ac.uk/ols4/ontologies/cl/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCL_0002319) is used by 30 different ontologies, for example, as a component term to define classes.

**Comment by bvarner-ebi** [comment]
@dosumis, given the concern noted in the comment above, would you still like to proceed with this obsoletion?

**Comment by dosumis** [comment]
> I think it causes problems if there is no replacement for CL:0002319.
[CL:0002319](https://www.ebi.ac.uk/ols4/ontologies/cl/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FCL_0002319) is used by 30 different ontologies, for example, as a component term to define classes.

I think we need to work out if it is being used directly by any ontologies, or whether it just turns up in import modules.

**Comment by dosumis** [comment]
The only way I know to do this is right now is with OntoBee (We should ask for this functionality with the OLS API).

The majority of use is in imports, but looking through I can see at least a couple of uses: 

https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/CL_0002319

[3D neural cell structure determination assay](https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/OBI_0003123) equivalentClass : [3D cell structure determination assay](https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/OBI_0003122) and ( [has_specified_output](https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/OBI_0000299) some [3D structural organization datum](https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/OBI_0001155)) and ( [has_specified_output](https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/OBI_0000299) some [is about](https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/IAO_0000136) some [neural cell](https://ontobee.org/ontology/ECO?iri=http://purl.obolibrary.org/obo/CL_0002319))

https://ontobee.org/ontology/UBERON?iri=http://purl.obolibrary.org/obo/CL_0002319

[brain](https://ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/UBERON_0000955) subClassOf : [composed primarily of](https://ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/RO_0002473) some [neural cell](https://ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/CL_0002319)
[nervous system](https://ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/UBERON_0001016) subClassOf : [composed primarily of](https://ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/RO_0002473) some [neural cell](https://ontobee.org/ontology/CL?iri=http://purl.obolibrary.org/obo/CL_0002319)


Need more complete exploration for other direct uses.

We can choose to drop from Uberon.   We could recommend that ECO use "cell and 'part of' some brain"  in place of neural cell...

**Comment by balhoff** [comment]
Here are a few ontologies with direct uses in existential restrictions: https://api.triplydb.com/s/89q9DYLEQ

- obo:hp/hp-base.owl
- obo:oba/oba-base.owl
- obo:obi/obi-base.owl
- obo:uberon/uberon-base.owl

**Comment by jahilton** [comment]
'neural cell' seems to be a general term and thus, can be useful in grouping its descendants. I believe that removing this term would set a precedent throughout CL and require a very substantial overhaul. Just a few examples: 'kidney cell' [CL:1000497], 'motile cell' [CL:0000219], 'connective tissue cell' [CL:0002320]
Or is there something specific to 'neural cell' that I am missing relative to those others?

**Comment by ANiknejad** [comment]
adding my 2-cents thoughts here: maybe the question is if broadly defined cells can be used for single-cell transcriptomics cell type identification, when a cell type can not be defined but still we know coming from a specific tissue (brain, kidney, connective tissue). Can we safely report 'connective tissue cell' [CL:0002320] or 'kidney cell' [CL:1000497] or 'neural cell' [CL:0002319] for such unknown cell types ? are T cells and any other invading cells considered as part of this tissue too ?
CellXGene likes reporting such unknown cells as 'native cell' but I see that this class is also under discussion for obsoleting!

https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/4.0.0/schema.md#integration-metadata

**Comment by ANiknejad** [comment]
yes, thank you Bradley, I saw also the issue 622 in the chanzuckerberg github, talking about the 2 options to replace ‘native cell’
